docs/mirror-maintain-guides-how-to-validate-kusama.md

```diff
@@ -232,6 +193,10 @@ account.
 
 ## Set Session Keys
 
+> **Note:** The session keys are consensus critical, so if you are not sure if your node has the current session keys
+> that you made the `setKeys` transaction for then simply run through the process of generating and setting new ones
+> using the `rotateKeys` method below. Better safe than sorry!
```

Comment:
Actually we have these two RPC methods that check if you have keys: `hasKey` and `hasSessionKeys`

https://polkadot.js.org/api/substrate/rpc.html#haskey-publickey-bytes-keytype-text-bool